File: listas-logpp/lista 1/lista casa C/questao 3.c
#include <stdio.h>
#include <math.h>

int main(){
    float AP, LP, TP, TA, AA, LA, QP;

    printf("digite a altura da parede:");
    scanf("%f", &AP);
    printf("digite a largura da parede:");
    scanf("%f", &LP);
    printf("digite a altura do azulejo:");
    scanf("%f", &AA);
    printf("digite a largura do azulejo:");
    scanf("%f", &LA);

    TP = AP * LP;
    TA = AA * LA;
    QP = TP / TA;
    printf("serao necessarios:%.0f azulejos", QP);
}

File: listas de exercicio/listas 1/lista casa python/questao 3.py
AP = input('digite a altura da parede:')
LP = input('digite a largura da parede:')
AP = float(AP)
LP = float(LP)
TP = AP * LP
TP = float(TP)

AA = input('digite a altura do azulejo:')
LA = input('digite a largura do azulejo:')
AA = float(AA)
LA = float(LA)
TA = AA * LA
TA = float(TA)
QP = TP / TA
print(f'serao necessarios:{QP} azulejos')